react/package.json

```diff
@@ -6,9 +6,10 @@
   "dependencies": {
     "@vtex/css-handles": "^1.0.0",
     "classnames": "^2.2.6",
+    "query-string": "^6.11.1",
```

Comment:
You can remove `query-string` since you're not using it, that'll keep the package size down.